tests/invert_cases/additionalProperties/any_false.json

```diff
@@ -0,0 +1,8 @@
+{
+  "schema": {
+    "anyAdditionalProperties": false
```

Comment:
I believe this is meant to mean "there must exist an additional property that validates against: false". So the inverse would be simply `true`. This inversion should be invertible, so this one ought to be consistent with `any_true.json`, right?

Reply:
I was using this table for the inversion implementation and tests:
```
{“additionalProperties” : true} ->  {“anyAdditionalProperty”: false}
{“additionalProperties” : false} -> {“anyAdditionalProperty” : true}
{“anyAdditionalProperty”: false} -> {“additionalProperties” : true}
{“anyAdditionalProperty” : true} -> {“additionalProperties” : false}
```

The value `{“additionalProperties” : true}` is equivalent to `true` and the value `{“anyAdditionalProperty”: false}` is equivalent to `false`. I think the cleanest way to handle the implementation is to not simplify those values and to leave them as is.

Reply:
Does that make sense?

Reply:
Yes.